src/country_details.jsx

```diff
@@ -0,0 +1,102 @@
+import { useLocation } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { createSearchParams, useSearchParams } from "react-router-dom";
+import { useParams } from 'react-router-dom'
+import get from './utils/axios-configure'
+import axios from 'axios'
+import { margin, style, width } from "@mui/system";
+import TinyFlag from "tiny-flag";
+import { URL_FLAG, URL_COUNTRY } from "./utils/const"
+
+
+
+
+
+export default function Details() {
+    const [searchParams, setSearchParams] = useSearchParams();
+    // const getcode=searchParams.get("code");
+    const queryParams = new URLSearchParams(window.location.search);
+    const { code } = useParams();
+    const [capitals, setCapital] = useState([]);
+    const [phonecodes, setPhonecode] = useState([]);
+    const [currencys, setCurrency] = useState([]);
+    const [flags, setFlag] = useState([]);
+
+    const header = {
+        'Content-Type': 'application/json'
+    };

```

Comment:
Move this object inside the `src/utils/axios-configure.js` file, there is no need for it here